configure.ac

```diff
@@ -425,6 +450,7 @@ case "$enable_hwloc" in
       ;;
 esac
 
+
```

Comment:
No need for an extra blank here …

Reply:
The current overall style of configure.ac is to have two blank lines between unrelated blocks.